src/node.cc

```diff
@@ -3661,16 +3665,19 @@ static void ParseArgs(int* argc,
     if (debug_options.ParseOption(arg)) {
       // Done, consumed by DebugOptions::ParseOption().
     } else if (strcmp(arg, "--version") == 0 || strcmp(arg, "-v") == 0) {
+      DisallowInNodeopts(argv[0], is_nodeopt, arg);
```

Comment:
when you say you will add them back in later.  Do you mean add the reasons for the exclusion as comments in the code ? I think a comment in the code as well is good in terms of longer term history when people look and wonder why its not allowed.  I do see you have the comments for some of them.